tools/flub/prettier.config.cjs

```diff
@@ -0,0 +1,31 @@
+// prettier.config.js or .prettierrc.js
+module.exports = {
+    printWidth: 100,
```

Comment:
Isn't our linter requirement elsewhere 120?

Reply:
Yes, but setting prettier to that length isn't ideal, because it tries to make lines as long as possible in many cases. 100 was the value that worked best when experimenting with prettier elsewhere in the repo.

> Prettier’s [printWidth](https://prettier.io/docs/en/options.html#print-width) option does not work the same way. It is not the hard upper allowed line length limit. It is a way to say to Prettier roughly how long you’d like lines to be. Prettier will make both shorter and longer lines, but generally strive to meet the specified printWidth.